CHANGELOG.rst

```diff
@@ -1,4 +1,4 @@
-**3.0.15 - TBD/TBD/TBD**
+**3.0.15 - 10/24/24**
```

Comment:
Don't forget to include these CI changes!

Reply:
Yea I still need to remove the branch in the build yml too so once other PR is merged Ill make final updates.